test/jobs/dspace_publication_results_job_test.rb

```diff
@@ -26,7 +26,7 @@ def setup
                                       'SubmissionSource' => { string_value: 'ETD', data_type: 'String' } } },
 
               # success but thesis no longer has files locally
-              { message_id: 'id1a', receipt_handle: 'handle1a',
+              { message_id: 'id1b', receipt_handle: 'handle1a',
```

Comment:
I... lol. Great catch!

Reply:
Yeah, I felt pretty silly when I caught that. 😅 (Both that we had duplicate IDs, and that we chose such a weird, brittle naming pattern!)